lib/glific/contacts/contact.ex

```diff
@@ -66,6 +67,7 @@ defmodule Glific.Contacts.Contact do
           last_message_number: integer,
           last_message_at: :utc_datetime | nil,
           last_communication_at: :utc_datetime | nil,
+          flows_paused_at: :utc_datetime | nil,
```

Comment:
why are we creating a new field instead of just using wakeup time in the context? why should this be a contact field?